PersonDataRequest/PersonDataRequest/PictureCell/PictureCell.xib

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This xib is not useful anymore, I gues it should be removed

Reply:
Removed